test/utils/install.js

```diff
@@ -1,7 +1,10 @@
-const gotoOutstandingTasks = require('./goto-outstanding-tasks');
+const gotoTasks = require('./goto-tasks');
 const completeRichText = require('./complete-rich-text');
+const gotoPlaces = require('./goto-places');
 
 module.exports = () => {
-  browser.addCommand('gotoOutstandingTasks', gotoOutstandingTasks);
+  browser.addCommand('gotoOutstandingTasks', gotoTasks('Outstanding'));
```

Comment:
```suggestion
  browser.setTimeout({ implicit: 200 });
  browser.addCommand('gotoOutstandingTasks', gotoTasks('Outstanding'));
```

I'm completely hijacking your PR for my own purposes here, but it's easier than building a whole new branch for one line.

Reply:
hehe